### PR TITLE
Add initial content

### PR DESCRIPTION
**Pull request by siakaramalegos**
This is the result of the edits we made. Let me know if you have more changes to propose!

### CODE REVIEW

**Comment by siakaramalegos** [comment]
Oh, should I just make a google form for now for anonymous reporting? Glynnis can do a design for the new site in June, and once I develop it, I can add a form directly on our website.

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
-# codeofconduct
+# Code of Conduct
+
+## Purpose
+
+Front-End Party is dedicated to being inclusive to the largest number of people, with the most varied and diverse backgrounds possible. As such, we strive to provide a friendly, safe and welcoming environment for all, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, ethnicity, age, socioeconomic status, and religion (or lack thereof).
+
+This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+
+We invite all those who participate in Front-End Party to help us create safe and positive experiences for everyone.
```
Comment (mshwery) [inline README.md:9]: `FrontEndParty` for consistency throughout this doc

File: README.md [hunk 2]
```diff
@@ -1 +1,75 @@
...
+If you are subject to or witness unacceptable behavior, feel unsafe or unwelcome, or have any other concerns, please notify a community organizer as soon as possible by either directly contacting the organizers or filing an anonymous report:
+
+### Direct Reporting
+
+Directly contact one of the organizers in person or through Meetup’s messaging system:
+
+- Cameron Sampson https://www.meetup.com/FrontEndParty/members/52557152/ 
+- Steven Achord https://www.meetup.com/FrontEndParty/members/185213572/ 
+- Sia Karamalegos https://www.meetup.com/FrontEndParty/members/98819682/ 
+- Matt Shwery https://www.meetup.com/FrontEndParty/members/21058911/
```
Comment (mshwery) [inline README.md:61]: Perhaps rather than keeping the community organizers listed here we can point to a Meetup link that displays all of them?

**Review by mshwery — APPROVED** [review]
Nice addition! We should consider adding a link to this on the website and from the Meetup page.

**Review by csampson — APPROVED** [review]

**Pull request merged**